cli/utils/runcmd.go

```diff
@@ -153,6 +153,15 @@ func CheckKubectl(verbose bool) bool {
 }
 
 func CheckMinikube(verbose bool) bool {
+	// on Windows, add the known Windows installation path to env:PATH to avoid minikube checking error
+	osName := runtime.GOOS
+	if strings.EqualFold(osName, "windows") {
+		var des = filepath.Join(os.Getenv("programfiles"), "maestro", "minikube")
+		path := AddtoPath(des)
```

Comment:
Will this cause the same path to be added to PATH multiple times during multiple runs?

Reply:
No, AddtoPath will check whether des is already contained in path and only add it when absence.

```go
func AddtoPath(des string) string {
	path := os.Getenv("PATH")
	pathlist := strings.Split(path, ";")
	for _, p := range pathlist {
		if strings.EqualFold(p, des) {
			return path
		}
	}
	return path + ";" + des
}
```